periphery/foundry.toml

```diff
@@ -7,6 +7,7 @@ libs = ['lib']
 out = '../build/periphery'
 cache_path = '../cache/periphery'
 
+via_ir = true
```

Comment:
I thought we were shying away from this - why are we using this now?

Reply:
After discussing it with you, I went and actually tried it out for the first time in a while. My primary concern was that it was still unstable, but I read through recent Solidity changelogs and it seems like in newer versions, there are actually _more_ bugs with the _old_ compilation pipeline than with the IR one. Plus it saved more gas than I expected.

So going forward, my thoughts are:
- Do our best to make things compile with `via_ir = false`, since it's nice to be able to go that route for quick testing (it compiles much much faster)
- Use `via_ir = true` in prod

Reply:
We were, but it has been updated quite a bit and we tested it again recently and found it helps us optimize a bit.

Reply:
Good to know